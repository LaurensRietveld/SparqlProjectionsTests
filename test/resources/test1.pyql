from pythonql.Executor import *

inData = results
programResult = { "cities": [
                  select {name: label}
                  for city in inData['results']['bindings']
                  let name = city['city']['value']
                  let label = city['cityLabel']['value']
                ]}
