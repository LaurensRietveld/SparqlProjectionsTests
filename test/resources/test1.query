SELECT ?city ?cityLabel WHERE {
  ?city rdf:type dbo:City .
  ?city rdfs:label ?cityLabel
}
LIMIT 5
