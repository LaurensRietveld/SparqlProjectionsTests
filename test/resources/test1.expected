{
  "cities": [{
      "http://dbpedia.org/resource/Amsterdam": "Amsterdam"
    }]
}
